docs/content/docs/3.recipes/3.prisma.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Prisma ORM
+description: Learn how to setup Prisma ORM with NuxtHub.
+---
+
+::alert{icon="i-ph-warning" color="red" title="WIP"}
+This ORM is not fully compatible with NuxtHub yet.
+:br
+- [Migrations](#migrations) are not supported yet.
```

Comment:
Migrations with D1 are not native with Prisma, [but do work](https://www.prisma.io/docs/orm/overview/databases/cloudflare-d1#migration-workflows)